### ISSUE TITLE
V0.15.0 `typescript-resolvers` plugin generates non-existing Resolver interfaces with noNamespaces

### ISSUE THREAD

**Issue by Rachnerd** [dotansimha/graphql-code-generator#1031]
**Describe the bug**
After upgrading to V0.15.0 and adding a workaround for [this issue](https://github.com/dotansimha/graphql-code-generator/issues/1029) the generated code does not compile anymore because IResolvers contains references to non-existing interfaces.

**To Reproduce**
1. clone https://github.com/Rachnerd/graphql-code-generator-issue/tree/resolver-issue
2. npm i && npm run generate
3. Inspect src/models starting at line: 123.

Used config: 
```yaml
schema: src/**/*.graphql
overwrite: true
generates:
  ./src/models.ts:
    config:
      noNamespaces: true
      interfacePrefix: GQL
    plugins:
      - typescript-common
      - typescript-server
      - typescript-resolvers
```

**Expected behavior**
I expect everything to compile.

**Environment:**
 - OS: Ubuntu 18.04
 - Codegen: 0.15.0
 - Node: v10.8.0

**Additional context**
I found this issue after upgrading from V0.14.4 to V0.15.0 without changing any configurations or code.

**Comment by ardatan** [comment]
@Rachnerd 
Could you try with canary version `0.16.0-alpha.af0870a1`?

**Comment by Rachnerd** [comment]
@ardatan 
`0.16.0-alpha.af0870a1` seems to fix it for both our projects ;)

**Comment by SimenB** [comment]
What's the timeline for a patch release of this? 🙂

**Comment by dotansimha** [comment]
@Rachnerd @SimenB released 0.15.1 :)
